src/components/MainSidebar/ProgramDimensionsPanel/useProgramDimensions.js

```diff
@@ -55,6 +61,11 @@ const reducer = (state, action) => {
                 error: action.payload,
                 dimensions: null,
             }
+        case ACTIONS.SET_LIST_END_VISIBLE:
```

Comment:
Same as we discussed the other day, is `ACTIONS` ever used in a standalone fashion? Or could these props just be extract to individual consts e.g. `const ACTION_RESET = 'RESET'`?